components/RefinementList/RefinementList.js

```diff
@@ -35,7 +35,9 @@ class RefinementList extends React.Component {
       [this.props.cssClasses.active]: facetValue.isRefined
     });
 
-    let key = facetValue[this.props.facetNameKey] + '/' + facetValue.isRefined + '/' + facetValue.count;
+    let key = facetValue[this.props.facetNameKey] + '/' + facetValue.isRefined;
+    if (facetValue.count !== undefined)
+      key += '/' + facetValue.count;
```

Comment:
I guess you should update the test to reflect this modification (that the cause of the problem right now). What about doing the same for the isRefined property?